db/migrations/20220929155704-create-tables.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    /**
```

Comment:
Oh is this file unused? We can remove it if it is

Reply:
Good catch, I don't think its in use. Not sure why this file was created, but we should be able to remove it.